tasks/contract.ts

```diff
@@ -66,8 +68,8 @@ async function list({}: { address: string }, hre: HardhatRuntimeEnvironment) {
 }
 
 task("generate", "generate a valid proof for an x,y pair")
-  .addParam("x", "x value", undefined, types.int)
-  .addParam("y", "y value", undefined, types.int)
+  .addParam("x", "x value", undefined, types.string)
+  .addParam("y", "y value", undefined, types.string)
   .setAction(generate);
 
 async function generate({ x, y }: { x: number; y: number }, hre: HardhatRuntimeEnvironment) {
```

Comment:
```suggestion
async function generate({ x, y }: { x: string; y: string }, hre: HardhatRuntimeEnvironment) {
```

Reply:
I think this might cause type errors with the `x.toString()` and `y.toString()` calls?